src/components/fieldset/fieldset-examples.mdx

```diff
@@ -0,0 +1,38 @@
+import { Meta, Canvas } from "@storybook/addon-docs";
+
+import * as stories from "./fieldset-examples.stories";
+
+<Meta
+  title="Fieldset/Address Fieldset Examples"
+  parameters={{ info: { disable: true } }}
+/>
```

Comment:
```suggestion
  of={stories}
  name="Fieldset Examples"
/>
```

**suggestion**: without this I get an error message when I go to `http://localhost:9001/?path=/docs/fieldset-address-fieldset-examples--docs`, complaining precisely that the `of` prop is missing.

The `name` prop is also necessary (without changing that there is a new error message suggestiting to set that - I make no claims that the name I've chosen is the best or most appropriate one!).

I assume we might also want to rename the import to something like `FieldsetExampleStories` as we have used similar naming conventions elsewhere - but that doesn't matter so much.